src/components/atoms/AchievementComponent/AchievementComponent.module.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../../foundations/sass/all";
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  width: fit-content;
+  justify-content: start;
+  margin: 1rem;
+}
+
+.info-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.main-content {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+}
+
+.image {
+  aspect-ratio: 1/1;
+  object-fit: cover;
+  padding: map-get($spacing, sm);
```

Comment:
Could you change `map-get` to `map.get` (Please use `@use "sass:map";` at the top of the file)